tests/autopilot_pool_expand/aut_pool_expand_test.go

```diff
@@ -0,0 +1,247 @@
+				},
+			},
+		},
+		Actions: []*apapi.RuleAction{
+			{
+				Name: portworx.StorageSpecAction,
+				Params: map[string]string{
+					portworx.RuleActionsScalePercentage: "50",
+					portworx.RuleScaleType:              portworx.RuleScaleTypeAddDisk,
+				},
+			},
+		},
+	},
+}
+
+// Resize storage pool by 50% until storage available capacity is less than %70
+var ruleResizeBy50IfPoolAvailableUsageMoreThan70ResizeDisk = apapi.AutopilotRule{
+	ObjectMeta: meta_v1.ObjectMeta{
+		Name: "storage-pool-resize-disk-70",
+	},
+	Spec: apapi.AutopilotRuleSpec{
+		Selector: apapi.RuleObjectSelector{
+			LabelSelector: meta_v1.LabelSelector{
+				MatchLabels: map[string]string{
+					"name": "storage-pool-resize",
+				},
+			},
+		},
+		Conditions: apapi.RuleConditions{
+			Expressions: []*apapi.LabelSelectorRequirement{
+				{
+					Key:      portworx.PxPoolAvailableCapacityMetric,
+					Operator: apapi.LabelSelectorOpLt,
+					Values:   []string{"70"},
+				},
+			},
+		},
+		Actions: []*apapi.RuleAction{
+			{
+				Name: portworx.StorageSpecAction,
+				Params: map[string]string{
+					portworx.RuleActionsScalePercentage: "50",
+					portworx.RuleScaleType:              portworx.RuleScaleTypeResizeDisk,
+				},
+			},
+		},
+	},
+}
+
+var ruleResizeBy50IfPoolTotalSizeLessThan300ResizeDisk = apapi.AutopilotRule{
+	ObjectMeta: meta_v1.ObjectMeta{
+		Name: "storage-pool-resize-disk-300",
+	},
+	Spec: apapi.AutopilotRuleSpec{
+		Selector: apapi.RuleObjectSelector{
+			LabelSelector: meta_v1.LabelSelector{
+				MatchLabels: map[string]string{
+					"name": "storage-pool-resize",
+				},
+			},
+		},
+		Conditions: apapi.RuleConditions{
+			Expressions: []*apapi.LabelSelectorRequirement{
+				{
+					Key:      portworx.PxPoolTotalCapacityMetric,
+					Operator: apapi.LabelSelectorOpLt,
+					Values:   []string{"300"},
+				},
+			},
+		},
+		Actions: []*apapi.RuleAction{
+			{
+				Name: portworx.StorageSpecAction,
+				Params: map[string]string{
+					portworx.RuleActionsScalePercentage: "50",
+					portworx.RuleScaleType:              portworx.RuleScaleTypeResizeDisk,
+				},
+			},
+		},
+	},
+}
+
+var autopilotruleBasicTestCases = []apapi.AutopilotRule{
+	ruleResizeBy50IfPoolAvailableUsageMoreThan70AddDisk,
+	ruleResizeBy50IfPoolAvailableUsageMoreThan70ResizeDisk,
+}
+
+func TestAutoPilot(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_autopilot.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Autopilot", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+// This testsuite is used for performing basic scenarios with Autopilot rules where it
+// schedules apps and wait until workload is completed on the volumes and then validates
+// storage pool(s) size(s) on the nodes where volumes reside
+var _ = Describe(fmt.Sprintf("{%sBasicAddDisk}", testNameSuite), func() {
```

Comment:
incorrect name "sBasicAddDisk". The `autopilotruleBasicTestCases` you use here also has a resize disk test case